c/meterpreter/source/common/config.h

```diff
@@ -43,6 +46,16 @@ typedef struct _MetsrvTransportCommon
 	int retry_wait;                       ///! Seconds to wait between reconnects.
 } MetsrvTransportCommon;
 
+//typedef struct _MetsrvTransportCommonDns
```

Comment:
This structure should be deleted as not used anymore.

Reply:
This PR is already pointing to the defcon-russia:master_update branch, so feel free to push these changes to that branch, and it will update the PR here.

Reply:
Done.